projects/ad4696_fmc/README.MD

```diff
@@ -0,0 +1,12 @@
+# HDL Reference Design for EVAL-AD4696FMCZ evaluation board
+
+## Product page
+
+change url
+http://www.analog.com/en/design-center/evaluation-hardware-and-software/evaluation-boards-kits/EVAL-AD40XX-FMCZ.html
```

Comment:
We should update this link, if we can, or delete it.

Reply:
that's why i wrote "change url" above the link :)

Reply:
We should not commit this kind of stuff to upstream, please delete the README if it does not contain any useful info.